lib/board.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class Board
+  attr_reader :board, :avail_squares
+  attr_writer :board, :avail_squares
+
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    @avail_squares = board.dup
+  end
+
+  def display_board
+    puts ' '
+    puts " #{board[0]}  |  #{board[1]}  |  #{board[2]}"
+    puts '----|-----|-----'
+    puts " #{board[3]}  |  #{board[4]}  |  #{board[5]}"
+    puts '----|-----|-----'
+    puts " #{board[6]}  |  #{board[7]}  |  #{board[8]}"
+    puts ' '
+  end
```

Comment:
You should not have `puts` or `print` or 'gets.chomp` anywhere outside the `main.rb` file.

Consider returning the string and then printing it out inside the `main.rb` file.

Reply:
I'm sorry mr ebukaume, I forgot to push the changes before, it's all good now.

thanks 👯‍♂

Reply:
Okay, no problem.